vignettes/vignette.Rmd

```diff
@@ -195,62 +218,65 @@ Let's revisit our country example from Rule 5. Does our list of country codes co
 sum(is.na(fossils$cc...36))
 ```
 
-Our test tells us that two of the values are `NA`: when we asked R to sort the values, it removed `NA` from the list of unique countries. However, the PBDB has set the country within which the collection is located as a compulsory entry field. How can we have missing data here? We don't: these values are not `NA`, they are "NA" meaning Namibia, and have been misconstrued by R. This is an important illustration of why we should conduct further investigation should any apparent errors arise in the dataset, rather than impulsively removing these data points.
+Our test tells us that two of the values are `NA`: when we asked R to sort the values, it removed `NA` from the list of unique countries. However, the PBDB has set the country within which the collection is located as a compulsory entry field. How can we have missing data here? We don't: these values are not `NA`, they are "NA" meaning Namibia, and have been misconstrued by R. This is an important illustration of why we should conduct further investigation when any apparent errors arise in the dataset, rather than immediately removing these data points.
 
 ## Rule 6: Identify outliers
 
-Next, we want to focus in on the specific variables which relate to our scientific question, i.e. the geography of our fossil occurrences. First we'll plot where the crocodile fossils have been found across the globe: how does this match what we already know from the country codes?
+Next, we want to look for outliers. Here we will focus in on the specific variables which relate to our scientific question, i.e. the geography of our fossil occurrences. First we'll plot where the crocodile fossils have been found across the globe: how does this match what we already know from the country codes?
 
 ```{r map}
 # Load in a world map
 world <- ne_countries(scale = "medium", returnclass = "sf")
+
 # Plot the geographic coordinates of each locality over the world map
 ggplot(fossils) +
   geom_sf(data = world) +
   geom_point(aes(x = lng, y = lat)) +
-  labs(x = "Longitude",
-       y = "Latitude") +
+  labs(x = "Longitude (º)",
+       y = "Latitude (º)") +
   coord_sf()
```

Comment:
If we're not defining any arguments, is this needed? I don't think it makes any difference as the relevant information (e.g. CRS) is pulled from geom_sf?